pages/pipelines/notifications.md.erb

```diff
@@ -80,9 +80,15 @@ The `basecamp_campfire` attribute accepts a single URL as a string.
 
 ## Slack Channel Message
 
-To send notifications to a Slack Channel, first set up the Slack notification service in your Organization's notification settings. Detailed instructions for setting up a Slack notification service can be found in the [Slack integration guide](/docs/integrations/slack). 
+Many teams find the messages generated by the [Slack integration configured with the buildkite.com web interface](/docs/integrations/slack) meet their needs. However, it's also possible to configure slack notifation within pipeline YAML to gain some extra configuration options.
 
-Add a Slack notification to your pipeline using the `slack` attribute of the `notify` yaml block:
+This requires Buildkite to have permission from Slack to message a channel. Configure the authorisation by following the [Slack integration guide](/docs/integrations/slack) for the desired channel. When asked to specify a pipeline, it's fine to select "Only some pipelines" and then check no pipelines.
+
+<section class="Docs__troubleshooting-note">
+  <p>Only Buildkite organisation administrators can authorize buildkite to message a slack channel.</p>
```

Comment:
```suggestion
  <p>Only Buildkite organization administrators can connect a Slack notification service. Once connected, any pipeline may post to it.</p>
```

But secondarily, should this be in the Slack doc instead of here?